src/test/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistrySpec.scala

```diff
@@ -21,7 +21,7 @@ class WalletRegistrySpec
     with WalletGenerators
     with FileUtils {
 
-  implicit override val generatorDrivenConfig = PropertyCheckConfiguration(minSuccessful = 5, sizeRange = 10)
+  implicit override val generatorDrivenConfig = PropertyCheckConfiguration(minSuccessful = 4, sizeRange = 10)
```

Comment:
It is too small for property testing. What is the reason to decrease it?

Reply:
With default value test is going to be slow and breaking Travis. The reason here is purely heuristic, more tests we have, less runs for a test we can have.